web-ui/src/app/components/tree-visualizer/tree-visualizer.component.scss

```diff
@@ -381,6 +365,30 @@
             box-shadow: 0 2px 8px rgba(0, 0, 0, 0.4);
             text-align: left;
 
+            .arrow {
+                position: relative;
+                top: 15px;
+                z-index:12;
+                box-sizing: border-box;
+                background: white;
+                box-shadow: 0px 3px 3px 0 rgba(0, 0, 0, 0.2);
+            }
+
+            .arrow::after{
```

Comment:
In scss you can combine this with the previous, so it would be:

```scss
.arrow {
  position: relative
  /* ... */

  &::after {
    content: "";
    /* ... */
  }
}
```